src/Sarif/Autogenerated/SarifNodeKind.cs

```diff
@@ -84,6 +80,14 @@ public enum SarifNodeKind
         /// </summary>
         Run,
         /// <summary>
+        /// A value indicating that the <see cref="ISarifNode" /> object is of type <see cref="Stack" />.
+        /// </summary>
+        Stack,
```

Comment:
I noticed the schema previously alphabetized these items and that I'd compromised sort order, so corrected.